unit-tests/test-phase.js

```diff
@@ -224,4 +225,23 @@ describe('phase', () => {
       assert.isFunction(phase.triggerGC);
     });
   });
+
+  describe('timeoutError', () => {
+
+    it('error thrown', () => {
+
+      let errorFunc = () => {
+        phase.timeoutError();
+      }
+
+      assert.throw(errorFunc, 'Test timeout exceeded ' + OPTIONS.timeout + 'ms');
```

Comment:
In my comment I didn't show the function being executed immediately, I was only passing a reference to the function. I am saying to use this:

`assert.throw(phase.timeoutError, ...)`

instead of:

`assert.throw(phase.timeoutError(), ...)`